src/test/kotlin/org/rust/ide/annotator/fixes/RemoveRedundantParenthesesFixTest.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import org.rust.ide.annotator.RsAnnotatorTestBase
+import org.rust.ide.annotator.RsExpressionAnnotator
+
+class RemoveRedundantParenthesesFixTest : RsAnnotatorTestBase(RsExpressionAnnotator::class) {
```

Comment:
Let's add some tests to ensure we don't provide warning annotation with quick fix for cases like `(1 + 2).pow(3)`